core/libraries/plugin_api/EE_Register_Privacy_Policy.lib.php

```diff
@@ -0,0 +1,75 @@
+<?php
+defined('EVENT_ESPRESSO_VERSION') || exit('No direct script access allowed');
```

Comment:
same here, we have to remove these from now on from files that contain classes.
They are ok to add to files that do not contain symbols (ie: files with code that executes upon loading)

Reply:
👍